src/Generator.cpp

```diff
@@ -29,7 +28,7 @@ namespace mia
 			.metavar("<std>")
 			.help("C++ standard to compile against")
 			.required(false).absent(CppStandard::Cpp11).nargs(1)
-			.choices({ "c++98", "c++03", "c++11", "c++14", "c++1z", "c++17", "c++2a", "c++20" })
+			.choices({ "c++98", "c++03", "c++11", "c++14", "c++1z", "c++17", "c++2a", "c++20" }) //TODO: get from enum
```

Comment:
TODO: implement before release

Reply:
It is not in scope for this PR. I just made sure that we will not miss it after implementing listing values for enums.